apps/etterlatte-behandling/src/main/kotlin/behandling/generellbehandling/GenerellBehandlingService.kt

```diff
@@ -133,21 +134,23 @@ class GenerellBehandlingService(
             null -> throw NotImplementedError("Ikke implementert")
         }
 
-        oppgaveService.ferdigStillOppgaveUnderBehandling(generellBehandling.id.toString(), saksbehandler)
+        val saksbehandlerNavn = saksbehandlerInfoDao.hentSaksbehandlerNavn(saksbehandler.ident)
         val trettiDagerFremITid = Tidspunkt.now().plus(30L, ChronoUnit.DAYS)
```

Comment:
Burde fristen for attestering være relatert til fristen for behandlingen? Har fristen for behandlingen gått ut, burde jo feks fristen for attestering være relativt kort vil jeg tro, slik at den prioriteres over andre ting med lang frist. Kanskje ikke relevant for denne PR'en, men la bare merke til det nå

Reply:
Burde nok justeres litt ja, men har latt den være som den er enn så lenge.